control-plane/subcommand/inject-connect/v2controllers.go

```diff
@@ -127,80 +127,80 @@ func (c *Command) configureV2Controllers(ctx context.Context, mgr manager.Manage
 		}
 	}
 
-	meshConfigReconciler := &controllersv2.MeshConfigController{
+	meshConfigReconciler := &controllersv2.ConsulResourceController{
```

Comment:
can we rename the var to `consulResourceReconciler` as well

Reply:
Sure thing. Is there a reason this is referred to as "Reconciler" instead of "Controller"?

Reply:
I gotcha, I'll change it to controller just for clarity.